domain/src/test/java/org/oppia/domain/oppialogger/analytics/AnalyticsControllerTest.kt

```diff
@@ -372,13 +408,111 @@ class AnalyticsControllerTest {
     assertThat(eventContext.revisionCardContext.subTopicId).matches(TEST_SUB_TOPIC_ID)
   }
 
+  @ExperimentalCoroutinesApi
+  @InternalCoroutinesApi
+  @Test
+  fun testController_logTransitionEvent_withNoneNetwork_checkLogsEventToStore() {
+    networkConnectionUtil.setCurrentConnectionStatus(NONE)
+    analyticsController.logTransitionEvent(
+      ApplicationProvider.getApplicationContext(),
+      TEST_TIMESTAMP,
+      EventAction.EVENT_ACTION_UNSPECIFIED,
+      analyticsController.createQuestionContext(
+        TEST_QUESTION_ID,
+        listOf(
+          TEST_SKILL_LIST_ID, TEST_SKILL_LIST_ID
+        )
+      )
+    )
+
+    val eventLogs = analyticsController.getEventLogs()
+    eventLogs.observeForever(this.mockOppiaEventLogsObserver)
+    testCoroutineDispatchers.advanceUntilIdle()
+    verify(
+      this.mockOppiaEventLogsObserver,
+      atLeastOnce()
+    ).onChanged(oppiaEventLogsResultCaptor.capture())
+
+    val eventLog = oppiaEventLogsResultCaptor.value.getOrThrow().getEventLog(0)
+    assertThat(eventLog.priority).isEqualTo(Priority.ESSENTIAL)
+    assertThat(eventLog.context.activityContextCase).isEqualTo(QUESTION_CONTEXT)
+    assertThat(eventLog.timestamp).isEqualTo(TEST_TIMESTAMP)
+    assertThat(eventLog.actionName).isEqualTo(EventAction.EVENT_ACTION_UNSPECIFIED)
+  }
+
+  @ExperimentalCoroutinesApi
+  @InternalCoroutinesApi
+  @Test
+  fun testController_logClickEvent_withNoneNetwork_checkLogsEventToStore() {
```

Comment:
logClickEvent_withNoNetwork instead of None maybe? (Here and elsewhere)

Reply:
Done.